recipes-sota/provision-device/provision-device_1.0.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Provision-device is a service to manually provision \
+the device to the Platform Services"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = " \
+    file://provision-device.sh \
+"
+
+S = "${WORKDIR}"
+
+RDEPENDS:${PN} = "bash jq curl unzip"
+
+inherit allarch
+
+do_install() {
+    install -d ${D}${sbindir}
+    install -m 0755 ${WORKDIR}/provision-device.sh ${D}${sbindir}
```

Comment:
should not be better to remove the `.sh` so we could call it only `provision-device`?

Reply:
Fixed